### ISSUE TITLE
The platform "darwin" is incompatible with this module.

### ISSUE THREAD

**Issue by pustovalov** [loeffel-io/ls-lint#8]
Hey, thx for this linter

I want to add your linter to the project, but in my team, people use both platforms: macOS and Linux

for example when I’m trying to install package for Linux on macOS:
```
[main]$ yarn add ls-lint-linux ls-lint-darwin --dev                                                                                                  
yarn add v1.22.4
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error ls-lint-linux@1.2.2: The platform "darwin" is incompatible with this module.
error Found incompatible module.

```

Maybe it is better to make one package that will load different binaries depending on the platform?

**Comment by loeffel-io** [comment]
Hey @pustovalov,

sorry for the inconveniences! 
**I will try to make this happen as soon as i have access to a windows machine 👍** 

The easiest solution should be to install the global package on your dev machines and using the linux package for your ci

**Comment by imlinus** [comment]
Hi,

I'm having similar problems when deploying to Netlify which uses Linux Arch. (and I'm using Darwin locally)

At the moment I'm using the global package instead 👍 (because Netlify doesn't need it)
I really like the idea of having one package that loads different versions based on platform

Cheers,
Linus

**Comment by loeffel-io** [comment]
Thanks @imlinus for your input! 

I will make that happen in the next 1-2 weeks!

**Comment by loeffel-io** [comment]
fyi: Docker support just released in v1.4.1 https://github.com/loeffel-io/ls-lint#docker

**Comment by imlinus** [comment]
You're a legend 🥇 Thank you for all your hard work 🙏
